durable_storage/src/lib.rs

```diff
@@ -22,3 +22,7 @@
 //!
 //! - **Persistence layer**: Responsible for actually persisting the basic get, set, delete
 //!   operations on disk.
+
+#![expect(dead_code, reason = "Stubbed API - will be implemented in RV-793")]
+mod persistence_layer;
```

Comment:
Would you mind adding a new line inbetween those? Otherwise it looks like the attribute applies to the module.

```suggestion
#![expect(dead_code, reason = "Stubbed API - will be implemented in RV-793")]

mod persistence_layer;
```